src/Game.js

```diff
@@ -0,0 +1,59 @@
+class Game {
+
+  constructor() {
+    this.currentFrame = new Frame();
+    this.scorer = new Scorer();
+  }
+
+  // User adds a roll to their scorecard
+  roll(pins) {
+    if (pins < 0 || pins > 10) {
+      throw "Please enter number between 0 and 10";
```

Comment:
0 and 10 appear multiple times across the code base. It would be cool to get rid of the [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)) and replace them with clearly-named constants

Reply:
Yes I should have seen this! Thanks for pointing it out.